gravwell.go

```diff
@@ -55,12 +55,13 @@ type encoder interface {
 	Name() string
 }
 
-func parseConfig(c *caddy.Controller) (conf cfgType, tag string, enc encoder, err error) {
+func parseConfig(c *caddy.Controller) (conf cfgType, enc encoder, err error) {
 	conf.IngestConfig = config.IngestConfig{
 		Log_Level:                `INFO`,
 		Ingester_Name:            `coredns`,
 		Insecure_Skip_TLS_Verify: false,
 	}
+	conf.Tag = defaultTag
```

Comment:
this is redundant given the other check below

Reply:
removed. PTAL